File: W1D2/fizzBuzz.js
const a = parseInt(process.argv[2]);
const b = parseInt(process.argv[3]);

for (let number = 1; number <= 100; number++) {
    if (number % a === 0 && number % b === 0) {
      console.log("FizzBuzz");
    } else if (number % b === 0) {
      console.log("Buzz");
    } else if (number % a === 0) {
      console.log("Fizz");
    } else {
      console.log(number);
    }
  }

File: W5D1/ruby_basics/fizz_buzz.rb
for number in 1..100
    if number % 3 == 0 && number % 5 == 0
        puts "FizzBuzz"
    elsif number % 5 == 0
        puts "Buzz"
    elsif number % 3 == 0
        puts "Fizz"
    else puts number
    end
end

# alt. way to achieve FIZZBUZZ

for number in 1..100
    str = ''
    str += "Fizz" if number % 3 == 0
    str += "Buzz" if number % 5 == 0
    if str.empty?
        puts number
    else
        puts str
    end
end